jerry-core/ecma/builtin-objects/ecma-builtin-internal-routines-template.inc.h

```diff
@@ -205,3 +223,5 @@ DISPATCH_ROUTINE_ROUTINE_NAME (uint16_t builtin_routine_id, /**< built-in wide r
 #undef BUILTIN_UNDERSCORED_ID
 #undef BUILTIN_INC_HEADER_NAME
 #undef ECMA_BUILTIN_PROPERTY_NAME_INDEX
+
```

Comment:
That was actually intentional, since the last define is kind of a separate group. I will reorganize them alphabetically. That could solve this problem.